dev_docs/older/readme.html

```diff
@@ -93,7 +93,7 @@
 	a game on it! Please refer to our complete game playing guide to get you
 	started on this subject.
 
-	Go to the <a href="./doc/game_playing_guide.html">Game Playing Guide</a>
+	Go to the <a href="../game_playing_guide/">Game Playing Guide</a>
```

Comment:
It's not that important, because those links are relative.
If our baseurl was `/test/` e.g. the index page would be located at `triplea-game.org/test/index.html` realtive links still work because the "current directory" stays the samerelative to all the other documents.
Only when using absolute links e.g. `/downloads/` we need to append the baseurl so the link gets changed to `/test/downloads`